lib/absinthe/phase/schema/validation/object_must_implement_interfaces.ex

```diff
@@ -191,6 +191,18 @@ defmodule Absinthe.Phase.Schema.Validation.ObjectMustImplementInterfaces do
     :ok
   end
 
+  defp check_covariant(
+         %Blueprint.TypeReference.Name{name: iface_name},
+         %Blueprint.TypeReference.Name{name: type_name},
```

Comment:
Ahh, I suspect the difference between the SDL & DSL is that only SDL currently generates the `TypeReference` blueprint structs everywhere... DSL winds up with bare-atom identifiers

This is a related note from a while back: https://github.com/absinthe-graphql/absinthe/issues/783

Reply:
You're exactly right! Thats what I was seeing, just a bare atom in the case of the DSL.

Reply:
ooh nice find